### DISCUSSION TITLE
ebus adapter V5 3D cad (STEP, STL)

### DISCUSSION THREAD

**Discussion by louiecaulfield** [john30/ebusd#1033] [Q&A]
Any chance of having 3D CAD files of the ebus adapter V5?

Would like to design an enclosure, having a CAD file to start from would help a bit.

**Comment by Koky05** [comment]
Enclosure already exist for it:
[ebus adapter v5 (shield)](https://www.thingiverse.com/thing:6000890)

**Reply by louiecaulfield** [reply]
I came across it and don't like it as it exposes the headers. I want to make a closed enclosure. As dust proof as possible :-)

**Reply by Koky05** [reply]
OK. But you can use it as a base ad make neccessary changes.

**Reply by andrewt27** [reply]
maybe try to use small electric junction box. I bought one:
![IMG_9668 Large](https://github.com/john30/ebusd/assets/36766135/5e2d62bb-8835-4c0c-aaef-f32d81e9d80b)
![IMG_9667 Large](https://github.com/john30/ebusd/assets/36766135/6ac26cb2-5b95-4969-9429-85ac6d1deb5e)
